src/lib/utilities/twitch.ts

```diff
@@ -0,0 +1,153 @@
+import { none, Result, some, type Option } from '@sapphire/result';
```

Comment:
Hello you what. I made @skyra/twitch-helpers for a reason. Please use it :angery: